### DISCUSSION TITLE
Issues with Invert Colors after installing MonitorControl

### DISCUSSION THREAD

**Discussion by hernan1999** [MonitorControl/MonitorControl#1316] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

Dear Support Team,

The app works great with my built-in screen (M1, 14’’) connected to a Dell display, both in regular mode as well as in “invert colors” mode. I use the latter at night to not flood out little house with light!

The problem is that, after installing the software, if I switch to “invert colors” without having a second screen connected, now the colors on the built-in screen will get inverted, but the overall picture will be saturated as shown in the attached picture. Do you know how to solve this? I’m assuming it’s something with color profiles, but I don’t know how to deal with them.

Thanks and all the best,

Hernan

![IMG_6954](https://user-images.githubusercontent.com/5193905/194568306-1a568a9e-72ee-4b44-b81f-9ba18f3ef177.jpg)

### Steps to reproduce

See attached image

### Expected behavior

The colors should not get saturated when switching to "invert colors"

### Environment Information (please complete the following information)

```markdown
- macOS version: 12.5.1
- Mac model: M1 14''
- MonitorControl version: 4.1.0
- Monitor(s): Dell p2715q (though issue is with built-in monitor)
- Apple Silicon/M1 (yes or no): YES
```

**Comment by waydabber** [comment]
Hi there - you should enable `Avoid gamma table manipulation` and also `Combine hardware and software dimming` so the app does not do software dimming (which will be brightening instead of dimming when colors are inverted) but only hardware control.

If this does not work well, you can try [BetterDisplay](https://betterdisplay.pro) - this app will produce a similar issue, but it has more granular control over what kind of techniques are used for each display for dimming so you can set up a configuration that works better for an inverted screen.

**Comment by hernan1999** [comment]
Hi!Thank you for your message. I’ll check this out!Best,Hernan***************************************************Hernan GarciaAssociate ProfessorDepartment of Molecular & Cell BiologyDepartment of PhysicsUniversity of California at BerkeleyChan Zuckerberg Biohub 501 Weill Hall, #3200Berkeley, CA ***@***.*** a meeting: meetme.so/HernanGarcia***************************************************On Nov 14, 2022, at 2:41 AM, Istvan Toth ***@***.***> wrote:﻿
Hi there - you should enable Avoid gamma table manipulation and also Combine hardware and software dimming so the app does not do software dimming (which will be brightening instead of dimming when colors are inverted) but only hardware control.
If this does not work well, you can try BetterDisplay - this app will produce a similar issue, but it has more granular control over what kind of techniques are used for each display for dimming so you can set up a configuration that works better for an inverted screen.
